in/command.go

```diff
@@ -122,14 +122,27 @@ func (command *Command) Run(destinationDir string, request Request) (Response, e
 				return Response{}, err
 			}
 
+			if request.Params.UnpackInto != "" {
+				request.Params.Unpack = true
+			}
 			if request.Params.Unpack {
-				destinationPath := filepath.Join(destinationDir, path.Base(remotePath))
```

Comment:
Renamed `destinationPath` to `archiveFilePath` because there is a possibility that the archive file name is not the destination path. It could be a custom path